driver.go

```diff
@@ -59,13 +59,36 @@ func (l *lvmDriver) Create(req volume.Request) volume.Response {
 		return resp(err)
 	}
 
-	cmdArgs := []string{"-n", req.Name}
+	cmdArgs := []string{"-n", req.Name, "--setactivationskip", "n"}
+	snap, ok := req.Options["snapshot"]
+	isSnapshot := ok && snap != ""
+	isThinSnap := false
+	if isSnapshot {
+		if isThinSnap, err = isThinlyProvisioned(vgName, snap); err != nil {
+			l.logger.Err(fmt.Sprintf("error checking if origin volume is thin, proceeding as if it is not: %s", err))
```

Comment:
Why are we ignoring the error (and just logging it). If there is an error we should do `return resp(err)`. Any specific reason for this ?